docs/website/docs/command-reference/json-output.md

```diff
@@ -370,7 +370,7 @@ odo registry -o json
 ]
 ```
 
-Using the `--details` flag, you will also get information about the Devfile:
+Using the `--details` flag with `--devfile <name>`, you will also get information about the Devfile:
 
 ```shell
 odo registry --devfile java-springboot --details -o json
```

Comment:
Can you also update the sample output below to include the new `versions.commandGroups` fields?

Reply:
Whoops, I think we'll also need the same thing for the sample output of `odo registry -o json` above.